fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -56,11 +57,28 @@ def self.install_wwdr_certificate
       Dir.chdir('/tmp') do
         url = 'https://developer.apple.com/certificationauthority/AppleWWDRCA.cer'
         filename = File.basename(url)
-        `curl -O #{url} && security import #{filename} -k login.keychain`
+        keychain = wwdr_keychain
+        keychain.prepend("-k ") unless keychain.empty?
+        `curl -O #{url} && security import #{filename} #{keychain}`
         UI.user_error!("Could not install WWDR certificate") unless $?.success?
       end
     end
 
+    def self.wwdr_keychain
+      priority = [
+        "security list-keychains -d user",
+        "security default-keychain -d user"
+      ]
+      priority.each do |command|
+        keychains = Helper.backticks(command, print: $verbose).split("\n")
+        unless keychains.empty?
+          # Select first keychain name from returned keychains list
+          return keychains[0].strip.tr('"', '').split(File::SEPARATOR)[-1]
+        end
+        return ""
```

Comment:
With both of these `return` statements in this `each` block, I don't think there's any way for the second **security** command to get executed. Is that intended?